src/crud/crud_heatmap_connectivity.py

```diff
@@ -0,0 +1,98 @@
+from src.crud.crud_heatmap import CRUDHeatmapBase
+from src.schemas.heatmap import (
+    IHeatmapConnectivityActive,
+    ActiveRoutingHeatmapType,
+)
+from src.core.config import settings
+from src.schemas.job import JobStatusType
+from src.core.job import job_init, job_log, run_background_or_immediately
+from src.schemas.layer import IFeatureLayerToolCreate, FeatureGeometryType
+from src.schemas.toolbox_base import (
+    DefaultResultLayerName,
+)
+
+
+class CRUDHeatmapConnectivityBase(CRUDHeatmapBase):
+    TRAVELTIME_MATRIX_TABLE = {
+        ActiveRoutingHeatmapType.walking: "basic.traveltime_matrix_walking",
+        ActiveRoutingHeatmapType.bicycle: "basic.traveltime_matrix_bicycle",
+    }
+    H3_10_CELL_AREA_SQM = 11285.625000840355
```

Comment:
maybe you can compute this dynamically? There is a function for this.

Reply:
For the connectivity heatmap query to be performant, I'm avoiding unnesting the array column but simply calculating array length * avg h3_10 area (after filtering by max_traveltime).

The library function h3_cell_area() takes a specific h3index as input, which would require unnesting the array, adding a lot of overhead to the query.

What I can do is put the avg area function `((3 * SQRT(3) / 2) * POWER(h3_get_hexagon_edge_length_avg(10, 'm'), 2))` into the query. I had hardcoded the value to further reduce overhead, but maybe this is fine.